### PR TITLE
Bootstrap3

### PR DESCRIPTION
**Pull request by danhawkins**
Support for bootstrap3, works well on my setup, not sure you would want to merge to master

### CODE REVIEW

File: vendor/assets/javascripts/bootstrap-datetimepicker.js [hunk 1]
```diff
@@ -966,13 +966,18 @@
           var expanded = $parent.find('.collapse.in');
           var closed = $parent.find('.collapse:not(.in)');
 
+          console.log(expanded);
+          console.log(closed);
```
Comment (lubieniebieski) [inline vendor/assets/javascripts/bootstrap-datetimepicker.js]: I'm sure you don't want to put `console.log` in this file:)

Comment (danhawkins) [inline vendor/assets/javascripts/bootstrap-datetimepicker.js]: You’re absolutely right, I’ll remove that right away

Regards

## Danny Hawkins

danny.hawkins@gmail.com
mobile: +44 (0)7447 158411
skype: dannyhawkins81
jabber: danny.hawkins@gmail.com
linkedin: http://www.linkedin.com/in/dannyhawkins

**Comment by lubieniebieski** [comment]
I really appreciate your effort, but you've changed something inside the core plugin, which I don't actually maintain - I've just added rails wrapper around it...

it's plugin-related issue, please contribute directly to https://github.com/tarruda/bootstrap-datetimepicker/issues. mentioned repo is no longer maintained though:/

**Comment by danhawkins** [comment]
Ah OK, no worries.

Regards

## Danny Hawkins

danny.hawkins@gmail.com
mobile: +44 (0)7447 158411
skype: dannyhawkins81
jabber: danny.hawkins@gmail.com
linkedin: http://www.linkedin.com/in/dannyhawkins

**Comment by xdmx** [comment]
@lubieniebieski thank you for your work... since the official bootstrap-datetimepicker is not maintained anymore, what do you think about using eonasdan's fork instead? https://github.com/Eonasdan/bootstrap-datetimepicker

**Pull request closed without merging**